graphjet-core/src/main/java/com/twitter/graphjet/datastructures/Pair.java

```diff
@@ -1,4 +1,4 @@
-package com.twitter.graphjet.algorithms;
+package com.twitter.graphjet.datastructures;
```

Comment:
Sure. I will use the apache one. Originally I thought to add a new Pair here will have one less external dependency. 

I am using it to return (LongList(node), LongList(metadata)), and the LongList inside does not have auto-boxing.